test/DartTest.ts

```diff
@@ -275,6 +277,10 @@ describe('transpile to dart', () => {
       expectTranslate("`\\$a`").to.equal(" '''\\$a''' ;");
     });
 
+    it('escapes escape sequences', () => {
+      expectTranslate("`\\\\u1234`").to.equal(" '''\\\\u1234''' ;");
```

Comment:
but we still have a problem in the angular code, right? https://github.com/angular/angular/blob/master/modules/angular2/src/change_detection/parser/lexer.js#L375 doesn't have four backslashes in the file, it has two. Unless the problem is that we have to escape the input here in a way we don't when reading from a file?